test/fd.js

```diff
@@ -354,3 +354,40 @@ tape('fd random-access write fails', function (t) {
     })
   })
 })
+
+tape('fd parallel reads', function (t) {
+  t.plan(3 * 2 + 1)
+
+  const drive = create()
+  const ws = drive.createWriteStream('test')
+
+  ws.write('foo')
+  setImmediate(function () {
```

Comment:
Just curious why these all need to be setImmediate'd

Reply:
To ensure that the stream is flushed (since we run in memory here setImmediate ensures all nextticks have run to completetion)